src/libs/OptionsListUtils.js

```diff
@@ -300,6 +306,15 @@ function getOptions(reports, personalDetails, draftComments, activeReportID, {
         userToInvite.icons = [defaultAvatarForUserToInvite];
     }
 
+    if (!isInChronosBeta && searchValue === 'chronos@expensify.com') {
+        userToInvite = null;
+        return {
+            personalDetails: [],
+            recentReports: [],
+            userToInvite,
+        };
```

Comment:
Hmm wait why we care if they are even in the beta or not? Simple to just add another condition above like

```js
&& (searchValue !== 'chronos@expensify.com')
```

then just don't set the `userToInvite` if they are trying to find this guy?

Reply:
Sorry nvm ignore me I didn't read any of the description lol

Reply:
Ok well something like this in there somewhere would work

```js
|| (isInChronosBeta && searchValue === 'chronos@expensify.com')
```

Reply:
`||` didn't work but `&& (isInChronosBeta && searchValue === CONST.EMAIL.CHRONOS)` did.

Reply:
Okay, what ended up working was adding `&& (searchValue !== CONST.EMAIL.CHRONOS || isInChronosBeta)` to the end if that lengthy condition check.

Reply:
Nice! yeah I knew some combination would 😝